server/src/main/java/org/elasticsearch/search/aggregations/support/AggregationInspectionHelper.java

```diff
@@ -82,7 +82,7 @@ public static boolean hasValue(UnmappedSampler agg) {
         return false;
     }
 
-    public static boolean hasValue(InternalAdjacencyMatrix agg) {
+    public static boolean hasValue(MultiBucketsAggregation agg) {
```

Comment:
I've never really understood the point of this method. It seems like we could assert this in the tests if we wanted.

Reply:
Right, this is used in tests (`GeoGridAggregatorTestCase`) and sql (`MetricAggExtractor`). I think this should be moved to sql (looks like the biggest user of this class?) and the one usage in `GeoGridAggregatorTestCase` can be replaced with adding a helper method that does `agg.getBuckets().stream().anyMatch(bucket -> bucket.getDocCount() > 0)`.

Let's do this in a followup change?

Reply:
:+1: